src/main/java/com/opentable/kafka/util/OffsetMonitor.java

```diff
@@ -0,0 +1,137 @@
+package com.opentable.kafka.util;
+
+import java.io.Closeable;
+import java.time.Duration;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.common.PartitionInfo;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.serialization.Deserializer;
+import org.apache.kafka.common.serialization.Serdes;
+
+import kafka.admin.AdminClient;
+import scala.collection.JavaConverters;
+
+public class OffsetMonitor implements AutoCloseable, Closeable {
+    private static final Duration TIMEOUT = Duration.ofSeconds(30);
+
+    private final String consumerGroupId;
+    private final KafkaConsumer<byte[], byte[]> consumer;
+    private final AdminClient adminClient;
+
+    /**
+     * Namespace the consumer created by this class; for example, pass your application name.
+     */
+    public OffsetMonitor(final String consumerNamespace, final String brokerList) {
```

Comment:
brokerlist is a comma delimited seed list? Kind of feel its lazy not structuring, but..admittedly it would just get joined

Reply:
Yeah, I agree.